recipe-server/normandy/base/api/views.py

```diff
@@ -64,6 +64,7 @@ def get(self, request):
         return Response(ServiceInfoSerializer({
             'user': user,
             'peer_approval_enforced': settings.PEER_APPROVAL_ENFORCED,
+            'logout_url': settings.OIDC_LOGOUT_URL or '/control/logout',
```

Comment:
We should probably use `reverse` to get the fallback URL so we don't have to keep this updated.